lib/dom.js

```diff
@@ -500,6 +501,7 @@ declare class Node extends EventTarget {
   cloneNode(deep?: boolean): this;
   compareDocumentPosition(other: Node): number;
   contains(other: ?Node): boolean;
+  getRootNode(): Node;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Node/getRootNode

Can you add the optional object argument?

Reply:
`void | {composed: boolean}` is what you want, since it's either omitted or or an object with a `composed` field.

Reply:
Thanks, I'm not familiarized with flow, sorry.

Just to be sure, this: `getRootNode(options: void | {composed: boolean}): Node;` is the same than `getRootNode(options?: {composed: boolean}): Node;`

Reply:
No problem! Best way to learn is to write libdefs like this.

Those two types are equivalent to callers, yes. I'm not sure how our function subtyping handles optional arguments vs. void union arguments, but `getRootNode(options?: {composed: boolean}): Node;` is totally fine for what we need here.